sql/updates/world/3.3.5/9999_99_99_99_world_335.sql

```diff
@@ -0,0 +1,4 @@
+DELETE FROM `spell_script_names` WHERE `spell_id`= 52267;
```

Comment:
is this another unrelated spell than the one in "INSERT INTO" ?

Reply:
Yes, despawn is handled in `spell_deliver_stolen_horse`.